src/main/java/edu/hm/hafner/analysis/parser/YuiCompressorParser.java

```diff
@@ -31,17 +34,27 @@
      * Creates a new instance of {@code YuiCompressorParser}.
      */
     public YuiCompressorParser() {
-        super(YUI_COMPRESSOR_WARNING_PATTERN, true);
+        super(YUI_COMPRESSOR_WARNING_PATTERN);
     }
 
     @Override
-    protected Optional<Issue> createIssue(final Matcher matcher, final IssueBuilder builder) {
-        String messageHeader = matcher.group(1);
-        CategoryAndPriority categoryAndPriority = getCategoryAndPriority(messageHeader);
-        String messageDetails = matcher.group(2);
-        String message = messageHeader + " [" + messageDetails + "]";
-        return builder.setFileName("unknown.file").setLineStart(0).setCategory(categoryAndPriority.getCategory())
-                             .setMessage(message).setSeverity(categoryAndPriority.getPriority()).buildOptional();
+    protected Optional<Issue> createIssue(final Matcher matcher, final LookaheadStream lookahead,
+            final IssueBuilder builder) {
+        if (lookahead.hasNext()) {
+            String messageHeader = matcher.group(1);
+            String messageDetails = lookahead.next();
+
+            CategoryAndPriority categoryAndPriority = getCategoryAndPriority(messageHeader);
+
+            String message = messageHeader + " [" + messageDetails + "]";
+
+            return builder.setFileName("unknown.file").setLineStart(0).setCategory(categoryAndPriority.getCategory())
+                    .setMessage(message).setSeverity(categoryAndPriority.getPriority()).buildOptional();
+
+        }
+        else {
```

Comment:
Rather than throwing an exception we should better create a new `Issue` that has a message without details.